src/components/FilterBar/FilterBar.react.js

```diff
@@ -46,7 +46,7 @@ export class FilterBar extends React.Component {
             }
 
             {this.context.filterBarStore.isExportable() &&
-              <ExportResultsButton
+              <ExportResultsList
```

Comment:
add check here, if exportAllOptions  is true, show ExportResultsList, otherwise show ExportResultsButton

Reply:
@hanpeic https://github.com/rdytech/react-filterbar/pull/138/commits/d6444960a04ceb4eed95d604253b2a12a297fa8d